src/js/components/AppConfigJSONEditorComponent.jsx

```diff
@@ -54,12 +54,13 @@ var AppConfigJSONEditorComponent = React.createClass({
     return (
       <AceEditor
         editorProps={{$blockScrolling: true}}
-        height="495px"
+        height="100%"
         mode="json"
         onChange={this.handleUpdate}
         showGutter={false}
         showPrintMargin={false}
         theme="monokai"
+        showGutter={true}
```

Comment:
My real problem is that there's no changelog entry `Fixed all IE issues forever` ;)